contrib/move-string-resource.kts

```diff
@@ -86,49 +83,48 @@ fun openOrCreateDestinationResourceFile(resourceFile: File, source: String, dest
 }
 
 fun createDestinationDirectoryAndFile(destinationDirectory: File, destinationFile: File) {
-  destinationDirectory.mkdir()
+  destinationDirectory.mkdirs()
   val isNewFileCreated: Boolean = destinationFile.createNewFile()
   if (!isNewFileCreated) {
     System.err.println("Could not create resource file ${destinationFile.path}")
     System.exit(-1)
   }
-  val initialXMLResourceFileContent = "<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n" +
-    "<resources xmlns:tools=\"http://schemas.android.com/tools\" tools:ignore=\"DuplicateStrings\">\n" +
-    "</resources>"
+  val initialXMLResourceFileContent = """
```

Comment:
I usually tab these so the ide shows a line running between the `"""` pairs, so I generally start them on a new line after the `=`. Having yet to write a kotlin script myself I am not sure if the highlighting is quite up to scratch but I imagine it should be

I would also be in favour of this being either not being a variable or be declared as a val at the top of the file.
I think inline is fine really
```
 destinationFile.writeText(
"""
    <?xml version=\"1.0\" encoding=\"UTF-8\"?>
    <resources>
    </resources>
""".trimIdent()
)
```
is pretty self explanatory to me.

Reply:
@Frans-Lukas this is my only remaining comment

Reply:
@Frans-Lukas this comment is still outstanding

Reply:
oh! missed that. On it.

Reply:
@macgills Should be fixed now 👍

Reply:
Excellent, we will get to merge this as soon as the build passes